orderer/consensus/etcdraft/chain_test.go

```diff
@@ -2385,11 +2385,20 @@ var _ = Describe("Chain", func() {
 					var newLeader, remainingFollower *chain
 					var c2state raft.SoftState
 					var c3state raft.SoftState
+					retry := 1
```

Comment:
I might be missing something, but, isn't `retry` always 1?

Reply:
Its mistake, suppose to decrement the value inside the if block. corrected it now.